llvm/lib/Target/RISCV/RISCVInstrInfoXqci.td

```diff
@@ -243,6 +253,24 @@ let Predicates = [HasVendorXqcilsm, IsRV32], DecoderNamespace = "Xqcilsm" in {
     def QC_LWMI : QCILoadMultiple<0b01, uimm5nonzero, "qc.lwmi">;
 } // Predicates = [HasVendorXqcilsm, IsRV32], DecoderNamespace = "Xqcilsm"
 
+let Predicates = [HasVendorXqcicli, IsRV32], DecoderNamespace = "Xqcicli" in {
+let hasSideEffects = 0, mayLoad = 0, mayStore = 0 in {
```

Comment:
Can we put the `hasSideEffects = 0, mayLoad = 0, mayStore = 0` on the `QCILICC` class? Like was done for `QCIStoreMultiple`, `QCISELECTIICC`, etc?